_examples/maps/maps.go

```diff
@@ -4,11 +4,19 @@
 
 package maps
 
-func MapsFunc(t map[string]int) {
+import (
+	"bytes"
+)
 
+func MapsFunc(t map[int]string) string {
+	var buffer bytes.Buffer
+	for _, v := range t {
+		buffer.WriteString(v)
+	}
+	return buffer.String()
 }
 
-func MapsFunc2() map[int]string {
+func CreateMaps() map[int]string {
```

Comment:
as there is only one `map` being created and returned, I'd just call this `CreateMap`, or, better yet: `New`.